api/src/main/java/org/pmiops/workbench/api/OfflineRdrExportController.java

```diff
@@ -17,13 +17,18 @@
 import javax.inject.Provider;
 import org.pmiops.workbench.config.WorkbenchConfig;
 import org.pmiops.workbench.config.WorkbenchLocationConfigService;
+import org.pmiops.workbench.exceptions.ServerErrorException;
 import org.pmiops.workbench.rdr.RdrExportService;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RestController;
 
 /**
- * This offline process is responsible to daily sync up with RDR by creating/pushing multiple task
- * with eligible user Ids and workspace Ids to cloud task queue
+ * This offline process is responsible to daily sync up with RDR by creating/pushing multiple Cloud
+ * Task tasks with eligible user Ids and workspace Ids to cloud task queue.
+ *
+ * <p>None of the actual RDR communication occurs from within this cron job handler, so this cron
+ * may succeed even though the actual export tasks to RDR are failing. See
+ * CloudTaskRdrExportController for the handlers which do the actual RDR export work.
```

Comment:
So we need to trigger an alert on the tasks failing in the queue. I bet I can get that from logs somewhere, assuming the tasks live in our GCP-space.

Reply:
Yes – this will be an important metric to track & alert on too.